bindings/pyroot/inc/TTreeAsFlatMatrix.h

```diff
@@ -0,0 +1,43 @@
+#ifndef ROOT_TTreeAsFlatMatrix
```

Comment:
ROOT preamble is missing (copyright, authors, etc)

Reply:
Good point, I'll add one. Thanks!